OMEdit/OMEdit/OMEditGUI/Modeling/ModelWidgetContainer.cpp

```diff
@@ -149,6 +149,16 @@ GraphicsView::GraphicsView(StringHandler::ViewType viewType, ModelWidget *pModel
   mpBitmapShapeAnnotation = 0;
   createActions();
 }
+bool GraphicsView::isCreatingShape()
+{
+  return isCreatingConnection() ||
```

Comment:
Do not include `isCreatingConnection()` here as it has a special case with ESC key.

Reply:
Adjusted